jwt-bcp/draft-ietf-oauth-jwt-bcp.md

```diff
@@ -284,12 +294,13 @@ Therefore, applications MUST only allow the use of cryptographically current alg
 that meet the security requirements of the application.
 This set will vary over time as new algorithms are introduced
 and existing algorithms are deprecated due to discovered cryptographic weaknesses.
-Applications must therefore be designed to enable cryptographic agility.
+Applications MUST therefore be designed to enable cryptographic agility.
 
 That said, if a JWT is cryptographically protected by a transport layer, such as TLS
 using cryptographically current algorithms, there may be no need to apply another layer of
 cryptographic protections to the JWT.
 In such cases, the use of the "none" algorithm can be perfectly acceptable.
+The "none" algorithm should only be used when the JWT is cryptographically protected by other means.
```

Comment:
Maybe add: Libraries that generate JWTs SHOULD NOT have "none" as a default options.

Reply:
New text:

JWT libraries should not generate JWTs using "none" unless explicitly requested to do by the caller.